### PR TITLE
Receiving translation for OAG

### PR DESCRIPTION
**Pull request by eng-prod-CI-bot-okta**
OAG translation for the latest commit: https://github.com/okta/okta-help/commits/docs_translations_OAG

### CODE REVIEW

File: oag/ja-jp/content/topics/access-gateway/advanced-oag-policy-development.htm [hunk 1]
```diff
@@ -318,17 +318,17 @@ <h3>高度なポリシーを定義する</h3>
...
-                                                                <li value="5"><span class="uicontrol">[Edit（編集）]</span>をクリックします。<br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/access-gateway/admin-policy-guide-advanced.03.png" data-mc-popup-alt="Click the pencil to open a policy for edit."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="764" data-mc-height="123" src="../../resources/images/access-gateway/admin-policy-guide-advanced.03_thumb_220_220.png" alt="鉛筆アイコンをクリックして編集対象のポリシーを開きます。" tabindex="" id="imgimg_new" /></a></li>
+                                                                <li value="5"><span class="uicontrol">[編集]</span>をクリックします。<br /><a class="MCPopupThumbnailLink MCPopupThumbnailPopup" href="../../resources/images/access-gateway/admin-policy-guide-advanced.03.png" data-mc-popup-alt="Click the pencil to open a policy for edit."><img class="MCPopupThumbnail img imgthumbnail" data-mc-width="764" data-mc-height="123" src="../../resources/images/access-gateway/admin-policy-guide-advanced.03_thumb_220_220.png" alt="鉛筆アイコンをクリックして編集対象のポリシーを開きます。" tabindex="" id="imgimg_new" /></a></li>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/advanced-oag-policy-development.htm:323]: @pengyao-okta I found a regression here. This seems to be a one-off, there are several other topics in this PR that have the correct rendering of UI labels.

File: oag/ja-jp/content/topics/access-gateway/boot-strap-oag-aws-post-deploy.htm [hunk 2]
```diff
@@ -392,7 +392,7 @@ <h2><a name="Specify"></a>ホスト名を指定 </h2>
 パスワード： &lt;default-password&gt;</span>
                                                                     <p><span class="mc-variable okta-feature-names.Access_Gateway_CMD variable">Access Gateway 管理者コンソール</span>に初めてサインインする場合、デフォルトパスワードを変更する必要があります。<a href="admin-first-login-mgmt-console.htm" class="MCXref xref">「Access Gateway 管理者コンソールに初めてサインインする」</a>をご参照ください。 </p>
                                                                 </li>
-                                                                <li value="2"><span class="userinput">[5 - System（システム）</span>を入力します。<br /></li>
+                                                                <li value="2"><span class="userinput">[5 - システム</span>を入力します。<br /></li>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/boot-strap-oag-aws-post-deploy.htm:395]: @pengyao-okta Just curious why the translator removed the EN portion of the string here, but did not do the same on the line immediately following? We should be consistent.
Anyway, this is a `userinput` attribute, I don't think we provided specific bilingual guidance for this attr.

File: oag/ja-jp/content/topics/access-gateway/cert-workflow-upload.htm [hunk 3]
```diff
@@ -341,29 +326,23 @@ <h2><a name="Add_a_new_certificate"></a>証明書を追加する</h2>
...
                                                         <div>
-                                                            <h2><a name="Modify_An_Existing_Certificate"></a>既存の証明書を修正する</h2>
-                                                            <p>既存の証明書を修正する場合、次の3つのオプションが表示されます。</p>
+                                                            <h2><a name="Modify_An_Existing_Certificate"></a>証明書を修正する。</h2>
+                                                            <p>次のいずれかのコマンドを選択します。</p>
                                                             <ul>
-                                                                <li><span class="code">[d] </span> - 証明書を削除する </li>
-                                                                <li><span class="code">[u]</span> - 証明書を更新する</li>
-                                                                <li><span class="code">[x]</span> - 変更せずに戻る </li>
+                                                                <li><span class="uicontrol">d</span>：証明書を削除します。</li>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/cert-workflow-upload.htm:342]: UI control labels without the brackets... but these should be `userinput` attrs anyway.

File: oag/ja-jp/content/topics/access-gateway/command-line-console-reference-network.htm [hunk 4]
```diff
@@ -390,7 +390,7 @@ <h3><a name="Edit"></a>/etc/hostsファイルの編集</h3>
                                                                 <p><span class="uicontrol">c</span>を押してホストファイルへの変更を確定します。</p>
                                                             </li>
                                                             <li value="6">
-                                                                <p><span class="uicontrol">x</span>を押してネットワーキングのメインメニューに戻ります。</p>
+                                                                <p><span class="uicontrol">x</span>を押して<span class="uicontrol">ネットワーキング</span>のメインメニューに戻ります。</p>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/command-line-console-reference-network.htm:393]: @pengyao-okta A new UI control attr with monolingual rendering

File: oag/ja-jp/content/topics/access-gateway/command-line-console-reference-network.htm [hunk 5]
```diff
@@ -424,8 +424,8 @@ <h4>プロキシの設定</h4>
...
                                                             <li value="6">必要に応じてプロキシのパスワードを入力し、<span class="uicontrol">Enter</span>を押します。<br /></li>
                                                             <li value="7">プロキシをバイパスする必要があるホスト名を入力します（カンマ区切り）。</li>
-                                                            <li value="8"><span class="uicontrol">y</span>を押してプロキシ設定を確定するか、<span class="uicontrol">N</span>を押して処理を中断します。</li>
-                                                            <li value="9"><span class="uicontrol">改行キー</span>を押してプロキシメニューに戻り、<span class="uicontrol">改行キー</span>を再度押してネットワーキングのメインメニューに戻ります。</li>
+                                                            <li value="8">プロキシ設定を確定する場合は<span class="uicontrol">y</span>、処理を中断する場合は<span class="uicontrol">N</span>を押します。</li>
+                                                            <li value="9"><span class="uicontrol">Enter</span>を押してプロキシメニューに戻り、もう一度<span class="uicontrol">Enter</span>を押して<span class="uicontrol">［Networking（ネットワーキング）］</span>のメインメニューに戻ります。</li>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/command-line-console-reference-network.htm:428]: @pengyao-okta Previously UI label was JA only, the translator changed it to EN only.

File: oag/ja-jp/content/topics/access-gateway/configure-idp-okta.htm [hunk 6]
```diff
@@ -324,27 +363,56 @@ <h2><span class="mc-variable okta-feature-names.Access_Gateway variable">Access
...
-                                                        <li value="3"><span class="wintitle">[Identity Providers（IDプロバイダー）]</span>ペインをクリックします。</li>
-                                                        <li value="4"><span class="uicontrol">［+］</span>をクリックし、<span class="uicontrol">［OKTA］</span>を選択します。</li>
-                                                        <li value="5">以下の情報を入力します。
-								<ul><li><span class="uicontrol">名前</span>：IdPにとって意味のある名前を入力します（例えば、Okta IdP）。</li><li><span class="uicontrol">Okta Org</span>：org名（例えば、<span class="userinput">orgname.oktapreview.com</span>、<span class="userinput">orgname.okta.com</span>など）を入力します。</li><li><span class="uicontrol">Okta APIトークン</span>：Okta APIトークンを作成したら、Okta orgからコピーしたトークをペーストします。</li></ul></li>
+                                                        <li value="1">ブラウザーで<span class="mc-variable okta-feature-names.Access Gateway_UI variable">Access Gateway管理者UIコンソール</span>に移動し、管理者としてサインインします。 </li>
+                                                        <li value="2"><span class="wintitle">[設定]</span>タブを選択します。</li>
```
Comment (paulwallace-okta) [inline oag/ja-jp/content/topics/access-gateway/configure-idp-okta.htm:389]: @pengyao-okta here's another `wintitle` attribute that regressed. Interestingly, the next line has a `wintitle` attr that keeps the EN (JA) pattern.

File: oag/ja-jp/content/topics/access-gateway/ref-arch/kerb-simp.htm [hunk 7]
```diff
@@ -319,7 +319,7 @@ <h2>利点と欠点</h2>
                                                                         <li>フォールトトレランスを持たない</li>
                                                                         <li>負荷分散や高可用性の機能がない</li>
                                                                         <li>本番環境を意図していない</li>
-                                                                        <li>管理サーバーへのアクセスには、SSHポート22とHTTPSポート443を開く必要があります。詳細については、<a href="../about-oag-prereqs.htm" class="MCXref xref">「Access Gatewayデプロイメントの前提条件」</a>を参照してください。</li>
+                                                                        <li>管理サーバーへのアクセスには、SSHポート22とHTTPSポート443を開く必要があります。詳細については、「<a href="../about-oag-prereqs.htm" class="MCXref xref">Access Gatewayデプロイメントの前提条件</a>」を参照してください。</li>
```
Comment (emikhasyak-okta) [inline oag/ja-jp/content/topics/access-gateway/ref-arch/kerb-simp.htm:322]: changes from [''link,,](x) to ''[link](x),, - just confirming if this is intentional?

File: oag/ja-jp/content/topics/access-gateway/trouble-shooting-guide.htm [hunk 8]
```diff
@@ -369,10 +369,10 @@ <h2><a name="Jump_to_top_↑"></a>どのような問題が発生しましたか
...
                                                     <div class="noteOkta">
-                                                        <p class="noteContent">ログは、ダウンロードすることも、ログサーバーに転送することもできます。詳細については、<a href="task-admin-logging.htm" class="MCXref xref">「ロギングの管理」</a>をご参照ください。</p>
+                                                        <p class="noteContent">ログは、ダウンロードすることも、ログサーバーに転送することもできます。詳細については、「<a href="task-admin-logging.htm" class="MCXref xref">ロギングの管理</a>」を参照してください。</p>
                                                     </div>
                                                     <ol>
-                                                        <li value="1"><span class="mc-variable okta-feature-names.Access_Gateway_CMD variable">Access Gateway 管理者コンソール</span>にログインします。デフォルトの資格情報は次のとおりです。<ul><li>ユーザー名： oag-mgmt</li><li>パスワード：OktaMgmt@123</li></ul></li>
+                                                        <li value="1"><span class="mc-variable okta-feature-names.Access_Gateway_CMD variable">Access Gateway管理コンソール</span>にログインします。デフォルトの資格情報は次のとおりです。<ul><li>ユーザー名： oag-mgmt</li><li>パスワード：OktaMgmt@123</li></ul></li>
```
Comment (emikhasyak-okta) [inline oag/ja-jp/content/topics/access-gateway/trouble-shooting-guide.htm:375]: removes white space between en and ja part - is this intentional?

**Review by emikhasyak-okta — APPROVED** [review]
I left a couple minor questions, looks good otherwise! 🚢

File: oag/ja-jp/content/topics/access-gateway/ref-arch/kerb-simp.htm (hunk 7 above)
Comment (paulwallace-okta) [reply oag/ja-jp/content/topics/access-gateway/ref-arch/kerb-simp.htm:322]: I'm guessing the translators decided that anchor tags should be surrounded by these JA-style quotation marks.

File: oag/ja-jp/content/topics/access-gateway/trouble-shooting-guide.htm (hunk 8 above)
Comment (paulwallace-okta) [reply oag/ja-jp/content/topics/access-gateway/trouble-shooting-guide.htm:375]: The space is not necessary in a JA context

**Pull request merged**